### DISCUSSION TITLE
渲染网页pdf时文字出现乱码

### DISCUSSION THREAD

**Discussion by sch401** [F9y4ng/GreasyFork-Scripts#372] [问答专区(Question / Answer)]
渲染网页pdf时文字出现乱码，例如这个页面https://xxgk.buct.edu.cn/2023/1221/c2825a186940/page.htm

**Comment by F9y4ng** [comment]
乱码问题是由于9月份月更新新增的针对Canvas字体渲染的实验性功能引起的新问题，下个月更新中会增加Canvas渲染的开关。

可暂时通过在该站点禁用渲染来解决乱码问题。

由于需要更多站点与环境的测试，该修复会在下个月更新中发布。
待问题修复后，画布背景重影问题可以通过向**排除渲染的HTML标签**追加 `,.textLayer *` 来解决。

### 效果预览
![screenshot_2024-09-11_13-27-07](https://github.com/user-attachments/assets/f218f262-0429-4703-b078-1b71c04b18b1)

**Comment by F9y4ng** [comment]
请更新至 v2024.09.11.1 ，在脚本设置界面取消勾选 **渲染画布**，并添加以上排除渲染的文本后，保存为站点独享数据即可。

![not render](https://github.com/user-attachments/assets/9143dd76-8b21-41c0-aeb5-b54da7fc5805)
